doc/admin-guide/files/records.config.en.rst

```diff
@@ -3264,6 +3264,7 @@ Client-Related Configuration
 ----------------------------
 
 .. ts:cv:: CONFIG proxy.config.ssl.client.verify.server INT 0
+   :overridable:
```

Comment:
This also makes the configuration reloadable doesn't it? If not, can we not make it reloadable now?